### PR TITLE
[Fixtures] Ability to reuse a fixture in the same suite

### LINKED ISSUE #5401: [FixturesBundle] Allow to use the same fixtures multiple times in one suite
It will allow for the following scenario:
 - add 5 random, enabled admin users
 - add 50 random, 90% enabled / 10% disabled regular users

```yaml
sylius_fixtures:
    suites:
        suite:
            fixtures:
                # Add 5 random, enabled admin users 
                - user:
                    random: 5
                    prototype:
                        admin: true
                        enabled: true

                # Add 50 random, 90% enabled / 10% disabled regular users
                - user:
                    random: 50
                    prototype:
                        admin: false
```

We need to replace configuration from `'fixtures' => ['user' => ...]` with `'fixtures' => [['user' => ...], ['user' => ...]]` and enhance suite factory.

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | fixes #5401
| License         | MIT

TODO:
 - [x] Document the new feature (#5434)

TODO after merge:
 - Create an issue about logging executed suites (they should show fixture aliases, not the actual fixture names)

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml [hunk 1]
```diff
@@ -69,9 +69,14 @@ sylius_fixtures:
                     options:
                         random: 3
 
-                user:
+                regular_user:
+                    name: user
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml]: Maybe ``type`` or alias?

Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml]: I come up with the following naming:
 - `regular_user` is an alias
 - `user` is a fixture name

It is based on `FixtureInterface::getName()` method (and similar to `ListenerInterface::getName()`).

I'm not sure for `type` too, as IMO it assumes that there is some closed list of available choices.

Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml]: Makes sense!

**Comment by pjedrzejewski** [comment]
Thanks Kamil! 👍

**Pull request merged**